### PR TITLE
Added Photo/Video permissions in plist

### PR DESCRIPTION
**Pull request by ts-leojoseph-r**
# Description
* Added Camera and Microphone permissions in plist.
* This is required for File Upload feature in JS SDK Sample app

## Links
MINI-3004

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

File: Example/Info.plist [hunk 1]
```diff
@@ -73,5 +73,9 @@
 		<string>UIInterfaceOrientationPortraitUpsideDown</string>
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 	</array>
+	<key>NSCameraUsageDescription</key>
```
Comment (corycaywood) [inline Example/Info.plist:76]: Could you also update the User Guide with information about why the Host App might need to add these permissions?

Comment (ts-leojoseph-r) [reply Example/Info.plist:76]: Done

**Review by corycaywood — APPROVED** [review]
LGTM 👍

File: Example/Info.plist [hunk 2]
```diff
@@ -73,5 +73,9 @@
...
+	<string>This will let you to take Photos</string>
+	<key>NSMicrophoneUsageDescription</key>
+	<string>Need Microphone while recording video</string>
```
Comment (Climbatize) [inline Example/Info.plist:79]: `Needs` or `We need`

File: USERGUIDE.md [hunk 3]
```diff
@@ -723,6 +724,17 @@ And the Mini App will be loaded like the following scheme,
 
 ```mscheme.rakuten//miniapp/index.html?param1=value1&param2=value2```
 
+
+<a id="permissions-from-host-app"></a>
+
+### Permissions required from the Host app
+
+Mini app requires following set of Device permissions that needed to be added in the Info.plist,
```
Comment (Climbatize) [inline USERGUIDE.md:732]: Mini App SDK requires the host app to include the following set of device permissions into its Info.plist file:

File: USERGUIDE.md [hunk 4]
```diff
@@ -723,6 +724,17 @@ And the Mini App will be loaded like the following scheme,
...
+
+* [NSLocationAlwaysAndWhenInUseUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nslocationalwaysandwheninuseusagedescription) - Location permission required by Mini app to track/get the current location of the user
+* [NSCameraUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nscamerausagedescription) - Camera permission required by Mini app to take pictures
+* [NSMicrophoneUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nsmicrophoneusagedescription) - Microphone permission required by Mini app to record a video.
```
Comment (Climbatize) [inline USERGUIDE.md:736]: Do you think we could have a matrix there?
| Permission  | Description | Reason  |
|---|---|---|
| [NSLocationAlwaysAndWhenInUseUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nslocationalwaysandwheninuseusagedescription)  | Location permission  | required to track/get the current location of the user |
| [NSCameraUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nscamerausagedescription)  | Camera permission  | required to take pictures |
| [NSMicrophoneUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nsmicrophoneusagedescription)  | Microphone permission  | required to record a video |

**Review by Climbatize — APPROVED** [review]
LGTM

File: Example/Info.plist (hunk 2 above)
Comment (ts-leojoseph-r) [reply Example/Info.plist]: Done

File: USERGUIDE.md (hunk 3 above)
Comment (ts-leojoseph-r) [reply USERGUIDE.md]: Done

File: USERGUIDE.md (hunk 4 above)
Comment (ts-leojoseph-r) [reply USERGUIDE.md]: Done

**Pull request merged**